src/object/obj_rpc.h

```diff
@@ -235,6 +238,22 @@ CRT_RPC_DECLARE(obj_punch, DAOS_ISEQ_OBJ_PUNCH, DAOS_OSEQ_OBJ_PUNCH)
 
 CRT_RPC_DECLARE(obj_query_key, DAOS_ISEQ_OBJ_QUERY_KEY, DAOS_OSEQ_OBJ_QUERY_KEY)
 
+#define DAOS_ISEQ_OBJ_SYNC	/* input fields */		 \
+	((uuid_t)		(osi_co_hdl)		CRT_VAR) \
+	((uuid_t)		(osi_pool_uuid)		CRT_VAR) \
+	((uuid_t)		(osi_co_uuid)		CRT_VAR) \
+	((daos_unit_oid_t)	(osi_oid)		CRT_VAR) \
+	((uint64_t)		(osi_epoch)		CRT_VAR) \
+	((int32_t)		(osi_map_ver)		CRT_VAR) \
+	((uint32_t)		(osi_padding)		CRT_VAR)
+
+#define DAOS_OSEQ_OBJ_SYNC	/* output fields */		 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
Related macro definition is the general format for DAOS code.